common/wuanalysis/anawu.hpp

```diff
@@ -20,13 +20,17 @@
 
 #include "anacommon.hpp"
 
-struct WuAnalyseOptions
+typedef enum { wuOptionFirst=0, minInterestingTime=0, minInterestingCost, skewThreshold } WuOptions ;
```

Comment:
A more common pattern for enumerations is to have a common prefix or suffix, so it is clear they belong together.  These names do make the calling code easy to read (but slightly confusing because they look like variable name).  I think they should be moved out of this file though (to avoid potential name clashes) - which should be possible if the virtuals are removed below.